### PR TITLE
feature-loginPageHeader

### PR DESCRIPTION
**Pull request by satyajeetdhere7444**
Test URLs:

- https://feature-loginPageHeader--hdfc-bank--aemsites.hlx.page/content/forms/af/hdfc

### CODE REVIEW

**Review by kunal-adobe — APPROVED** [review]

File: styles/header-banner.css [hunk 1]
```diff
@@ -155,4 +210,16 @@ header {
...
+            div:nth-child(1){
+                text-align: center;
+            }
+
+            div:nth-child(2), div:nth-child(3),  div:nth-child(4),  div:nth-child(5){
+                display: none;
+            }
+        }
+    }
 }
```
Comment (srivastava21ashish) [inline styles/header-banner.css:225]: EOL missing

File: styles/header-banner.css [hunk 2]
```diff
@@ -41,43 +41,73 @@ header {
     background: var(--color-interface-background-tertiary);
```
Comment (srivastava21ashish) [inline styles/header-banner.css:41]: have to imported this file somewhere?

**Review by srivastava21ashish — CHANGES_REQUESTED** [review]

File: styles/header-banner.css (hunk 2 above)
Comment (satyajeetdhere7444) [reply styles/header-banner.css:41]: yes, this var declare in **color-palette.css** and it imported in styles.css, we are imporating all css file styles.css

File: styles/header-banner.css (hunk 1 above)
Comment (satyajeetdhere7444) [reply styles/header-banner.css:225]: it didn't show the lint issue

**Pull request merged**